public/app/features/library-panels/LibraryPanelsPage.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { FC, useState } from 'react';
+import { connect, ConnectedProps } from 'react-redux';
+
+import { GrafanaRouteComponentProps } from '../../core/navigation/types';
+import { StoreState } from '../../types';
+import { getNavModel } from '../../core/selectors/navModel';
+import Page from '../../core/components/Page/Page';
+import { LibraryPanelsView } from './components/LibraryPanelsView/LibraryPanelsView';
+import { useAsync } from 'react-use';
+import { getLibraryPanels } from './state/api';
+import OrgActionBar from '../../core/components/OrgActionBar/OrgActionBar';
+
+const mapStateToProps = (state: StoreState) => ({
+  navModel: getNavModel(state.navIndex, 'library-panels'),
+});
+
+const connector = connect(mapStateToProps, undefined);
+
+interface OwnProps extends GrafanaRouteComponentProps {}
+
+type Props = OwnProps & ConnectedProps<typeof connector>;
+
+export const LibraryPanelsPage: FC<Props> = ({ navModel }) => {
+  const [searchQuery, setSearchQuery] = useState('');
+  const { value: searchResult, loading } = useAsync(async () => {
+    return getLibraryPanels();
+  });
+  const hasLibraryPanels = Boolean(searchResult?.libraryPanels.length);
+
+  return (
+    <Page navModel={navModel}>
+      <Page.Contents isLoading={loading}>
+        {hasLibraryPanels && (
+          <OrgActionBar searchQuery={searchQuery} setSearchQuery={setSearchQuery} placeholder={'Search by name'} />
```

Comment:
Nit: maybe we an rename this component to sth that's not that much connected to Org concept?

Reply:
What about PageActionBar instead?

Reply:
Yes, i think this would work better. We could actually use the namespace pattern and have Page.ActionBar component. Maybe a separate PR to do that? :)

Reply:
Renamed it PageActionBar, did not fix the namespace in this PR.